kratos/processes/apply_multi_point_constraints_process.h

```diff
@@ -85,17 +91,26 @@ class ApplyMultipointConstraintsProcess : public Process
 
     ApplyMultipointConstraintsProcess(ModelPart &model_part, std::string name = "default") : Process(Flags()), mr_model_part(model_part), m_parameters("{}")
     {
+        Parameters default_parameters(R"(
+            {
+                "constraint_set_name":"default",
+                "master_sub_model_part_name":"default_master",
+                "slave_sub_model_part_name":"default_slave",                
+                "variable_names":[],
+                "reform_every_step":false   
+            }  )");
+        m_parameters.RecursivelyValidateAndAssignDefaults(default_parameters);
 
         // IMPORTANT : This constructor is not to be used when using this process in the normal KRATOS process_list of python script
         ProcessInfoPointerType info = mr_model_part.pGetProcessInfo();
-        if (info->GetValue(MPC_DATA_CONTAINER) == nullptr)
-            info->SetValue(MPC_DATA_CONTAINER, MpcDataSharedPointerVectorType(new std::vector<MpcDataPointerType>()));
+        if (info->GetValue(CONSTRAINTS_CONTAINER) == nullptr)
+            info->SetValue(CONSTRAINTS_CONTAINER, ConstraintSharedPointerVectorType(new std::vector<ConstraintPointerType>()));
```

Comment:
This is to facilitate  swapping the set of constraints which are available to builder and solver (later on) which out much data overhead. Having a pointer to this vector means one can at some point swap this pointer to a different one have a different set of constraints applied on the system. 

This can also be done by setting a Constraint to be active or inactive.